source/content/guides/composer-convert.md

```diff
@@ -25,34 +36,70 @@ export site=yoursitename
+You're about to make massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
 
 1. In your local terminal, change directories to your site project. For example, if you keep your projects in a folder called `projects` in the home directory:
 
   ```bash{promptUser:user}
   cd ~/projects/$site/
   ```
 
-2. Create the new branch:
+1. Add the Pantheon Drupal Upstream as a new remote, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
+  This upstream is a useful starting point for converting Drupal 8 sites to Composer, but it is a Drupal 9 upstream. To use it for Drupal 8, we will adjust the version numbers in a later step.
 
-## Set up a Multidev (Optional)
+  You can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
+
+1. Copy your configuration from the default branch:
+
+  ```bash{promptUser:user}
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git rm -f sites/default/config/.htaccess
+  git commit -m "Pull in configuration from default branch"
+  ```
+
+1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
+
+  ```bash{promptUser:user}
+  git diff master:pantheon.yml pantheon.upstream.yml
+  ```
+
+1. If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old pantheon.yml to preserve these settings:
+
+  ```bash{promptUser:user}
+  git checkout master pantheon.yml
+  git add pantheon.yml
+  git commit -m 'Copy my pantheon.yml'
+  ```
 
-If your Pantheon account has access to [Multidev](/multidev), create a Mmultidev to push your new code to:
+Note that a few settings must be left as they are in `pantheon.upstream.yml` for sites using Integrated Composer. These are:
+
+* `web_docroot`
+* `build_step`
+
+### (Optional) Return to Drupal 8
+
+If you’re not ready to update to Drupal 9 yet. (Advisable: test the Composer conversion before testing the Drupal 9 upgrade)
+Note that you might want to manually edit `composer.json` and remove the `drupal/core` entry if it exists. The empty drupal/core entry from the steps above will cause drupal/core to be erased and redownloaded frequently
 
 ```bash{promptUser:user}
-git push origin composify && terminus env:create $site.dev composify
+cd upstream-configuration/
+composer require drupal/core-recommended:^8 --no-update
```

Comment:
Note 1: this will be inaccurate once https://github.com/pantheon-systems/drupal-project/pull/6 is released

Note 2: due to a new development / release process we're putting into effect, merging that PR is not the same as it being released. I'm making a note to ping @EdwardAngert when it's actually released.